buildspec.yml

```diff
@@ -8,15 +8,17 @@ phases:
       - echo Running build for branch/pr/tag $CODEBUILD_WEBHOOK_TRIGGER
   build:
     commands:
+      - export timestamp=$(date +"%Y%m%dT%H%M%S")
       - export BUILD_TAG=$(echo $CODEBUILD_WEBHOOK_TRIGGER | sed 's/.*\///g')
       - echo Build started on `date`
       - echo Building the Docker image...          
       - docker build -t datapusher:$BUILD_TAG .
-      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG
+      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG.latest
+      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG.${timestamp}
   post_build:
     commands:
+      - export timestamp=$(date +"%Y%m%dT%H%M%S")
```

Comment:
Do we even need the `timestamp` and `BUILD_TAG` here after changing the way we push datapusher images in line :24?

Reply:
you're right, we don't :)